packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -199,13 +199,13 @@ describe('basic grid', () => {
     );
 
     // Find column headers
-    const nonSelectableCell = wrapper.find(Row).at(0).find('.selectable');
+    const nonSelectableCell = wrapper.find(Row).at(0).find('th:not(.selectable)');
```

Comment:
Why did this have to change in the test?

Reply:
We believe this line was inconsistent with the intention of the test and needed to be fixed. The intention of the test is to verify selection of a **non-selectable** cell in row selection mode. Though clicking both selectable & non-selectable cell will result in row-selection, we changed this line to be consistent with the intention of the test.

Reply:
Based on the example worklist data grid used by the test, if you don't use the _th_, the find will return many things such as _FocusTrap, div, span, td_ & _th_ that do not have the selectable class. Using the _th_ in the selector allows you to get to the non-selectable cell directly and makes the test cleaner.

Reply:
That could only happen if the selector is incorrect.  If you replaced th is the class associated with the th and td elements, then you would still get that element.  I can approve for now, and we can update later, if necessary.